src/prefect/runner/server.py

```diff
@@ -1,15 +1,25 @@
+import typing as t
```

Comment:
`t` is a bit too obscure, maybe we could use `_typing`

Reply:
alternatively, and probably the better option, let's import the objects we need explicitly